boards/nrf52840-dongle/Makefile.dep

```diff
@@ -0,0 +1,25 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+# It'd be nice to pull this in on any stdin dependency and not just shell, but
+# that's the common one, and by the time shell has pulled in stdin, the default
+# stdio_uart has already been set.
+#
+# Maybe have a shell_default like netdev_default?
+ifneq (,$(filter shell,$(USEMODULE)))
```

Comment:
What if there is no shell? Is there then no STDIO? I'm not sure we support that concept ^^" (see https://github.com/RIOT-OS/RIOT/pull/10741).